app/commands/filter_pull_requests.rb

```diff
@@ -2,6 +2,19 @@ class FilterPullRequests < PowerTypes::Command.new(:all_pull_requests, :options)
   def perform
     filter(:project_name) { |pr| pr.by_repository(@options[:project_name]) }
     filter(:owner_name) { |pr| pr.by_owner(@options[:owner_name]) }
+    filter(:title) { |pr| pr.by_title(@options[:title]) }
+    filter(:start_date) { |pr| pr.by_start_date(@options[:start_date]) }
+    filter(:end_date) { |pr| pr.by_end_date(@options[:end_date]) }
+    filter(:top_liked) { |pr| pr.by_top_liked.limit(15) }
+    # if false
+    #   # :top_liked.present?
+    #   print :top_liked
+    #   print "\n\n\nCHILEEE\n\n\n"
+    #   pull_requests
+    #   print pull_requests
+    #   # pull_requests = @pull_requests.all.includes(:likes).sort_by { |pr| pr.likes.count }.reverse
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [101/100]

Reply:
dejé estos comentarios que se me olvidó borrar, los borro ahora